styles.css

```diff
@@ -30,7 +30,7 @@ body {
 /* parent container */
 .container {
   background-color: #f7f7f9;
-  max-width: 375px;
+  width: 375px;
   margin: 0 auto;
```

Comment:
- [ ] Please know that fixing this width caused some issues in your design, now it's not responsive in the mobile version, please know that it should be responsive for almost all mobile devices meaning, from width `375px` to `768px`, in your case the mobile version is only available at 375px width, when you increase the width it will be like this:

![image](https://user-images.githubusercontent.com/20567503/179373775-f6927c7c-1854-477a-a674-5f38a2864998.png)

Kindly remove any fixed width you have, you should handle the containers with flexbox or grid to maintain flexibility. 😁